dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -33,7 +33,9 @@ module Pd::WorkshopSurveyResultsHelper
     :who_facilitated
   ]
 
-  FIELDS_IN_SUMMARY = MULTIPLE_CHOICE_FIELDS_IN_SUMMARY + FREE_RESPONSE_FIELDS_IN_SUMMARY
+  LOCAL_WORKSHOP_FIELDS_IN_SUMMARY = LOCAL_WORKSHOP_MULTIPLE_CHOICE_FIELDS_IN_SUMMARY + FREE_RESPONSE_FIELDS_IN_SUMMARY
+  TEACHERCON_MULTIPLE_CHOICE_FIELDS = Pd::TeacherconSurvey.public_required_fields & Pd::TeacherconSurvey.options.keys
+  TEACHERCON_FIELDS_IN_SUMMARY = Pd::TeacherconSurvey.public_required_fields + Pd::TeacherconSurvey.facilitator_required_fields
```

Comment:
Should these be using [Pd::TeacherconSurvey.public_fields](https://github.com/code-dot-org/code-dot-org/blob/c98b4a9ee21c4a93a93482cc74ba6bb098b56fb8/dashboard/app/models/pd/teachercon_survey.rb#L44) instead of `public_required_fields`?

nit: these constants should be frozen.